recipe_scrapers/giallozafferano.py

```diff
@@ -1,7 +1,38 @@
 from ._abstract import AbstractScraper
-
+import re
 
 class GialloZafferano(AbstractScraper):
     @classmethod
     def host(cls):
         return "ricette.giallozafferano.it"
+
+    def instructions(self):
+        instructions = self.schema.data.get("recipeInstructions", [])
+        if isinstance(instructions, list):
+            return "\n".join(self._normalize_string(instr) for instr in instructions if instr.strip())
+        return self._normalize_string(instructions)
+
+    def nutrients(self):
+        nutrients = self.schema.data.get("nutrition", {})
+        return {
+            "calories": self._fix_comma(nutrients.get("calories")),
+            "fatContent": self._fix_comma(nutrients.get("fatContent")),
+            "saturatedFatContent": self._fix_comma(nutrients.get("saturatedFatContent")),
+            "carbohydrateContent": self._fix_comma(nutrients.get("carbohydrateContent")),
+            "sugarContent": self._fix_comma(nutrients.get("sugarContent")),
+            "sodiumContent": self._fix_comma(nutrients.get("sodiumContent")),
+            "fiberContent": self._fix_comma(nutrients.get("fiberContent")),
+            "cholesterolContent": self._fix_comma(nutrients.get("cholesterolContent")),
+        }
```

Comment:
```suggestion
        nutrient_keys = [
            "calories",
            "fatContent",
            "saturatedFatContent",
            "carbohydrateContent",
            "sugarContent",
            "sodiumContent",
            "fiberContent",
            "cholesterolContent",
        ]
        return {key: self._fix_comma(nutrients.get(key)) for key in nutrient_keys}
        }
```

Instead of repeating through each key this can be looped for a cleaner run without any changes to functionality.